package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "sjrk-storytelling",
+  "version": "0.0.1",
+  "description": "SJRK Storytelling Project",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Looks like this project's tests are all browser-based, so we can remove this `"test"` script command.